microsoft-edge/devtools-guide-chromium/evaluate-performance/unminify.md

```diff
@@ -0,0 +1,106 @@
+
+Source mapping is used in DevTools to keep your client-side code readable and debuggable.  By clicking the **Unminify** button in the Performance tool, you can use source maps to create an unminified version of the performance profile you recorded.
+
+
+<!-- ====================================================================== -->
+## How to unminify a performance profile
+
+To unminify a performance profile, so that some more meaningful function names are shown in addition to the minified identifiers, follow the steps below.
+
+   **Set up source maps:**
+
+1. Set up source maps, to host your source maps side-by-side with your production code.  Host your source maps as well as your production code, where the **Performance** tool can find your source maps.<!-- Can you use ado server for secure hosting of source maps?-->  For details, see [Map the processed code to your original source code, for debugging](/microsoft-edge/devtools-guide-chromium/javascript/source-maps).
+
+   <!-- Or, if you are just trying out this feature, you can go to a browser-based Outlook mailbox: `https://outlook.office.com`, because this web app produces minified function names on the call stack. -->
+
+   In the following example, the source maps are served from the same server directory as the production code:
+
+   ![The source maps are served side-by-side with production code.](images/unminify-perf-profile-sourcemaps.msft.png)
+<!-- this was the best way I could come up with to describe the folder structure. https://docs.microsoft.com/en-us/dotnet/core/porting/project-structure is an example that went in a different direction but if you like it, please replace this image with a diagram -->
+
+   <!-- a text-based format that could replace the Sources > Page tab screenshot:
+   ```
+   hooks.ts
+   index.js
+   index.ts
+   isLoki
+   ```
+   -->
+
+   **Record a performance profile:**
+
+1. In the **Performance** tool, click the **Record** button, to record a performance profile for the page you want to analyze.  For details, see [Record runtime performance](/microsoft-edge/devtools-guide-chromium/evaluate-performance/reference).
+
+   <!-- If you are just quickly trying out the feature, you can record a performance profile while you scroll through a couple of emails and click them at your inbox at the Outlook web app. -->
+
+   ![Recording a performance profile.](../media/evaluate-performance-performance-record-highlight.msft.png)
+
+   A performance profile is recorded, in the **Performance** tool.
+
+1. In the **Performance** tool, click in the main area (the call stack), and use the mouse wheel to zoom in.  Click and drag left and right.  Look at the function names in the call stack.  Select a small section of the profile and look at the call stack.  Note the timestamp range.
+
+   The minified call stack in the **Performance** tool shows function names as **b**, **O**, **Xt**, **bn**, **n**, **i**, or **Yn**.
```

Comment:
It should be made clear that these function names are just examples of what minified names might look like.